src/components/LoadingPlaceholder.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export default function LoadingPlaceholder() {
+  return <PlaceholderContainer></PlaceholderContainer>;
+}
+
+const PlaceholderContainer = styled.div`
+  display: none;
+`;
```

Comment:
if this is not displayed, do you need it though?

Reply:
I haven't found a better solution yet, cause I need something for the conditional rendering, but don't want it to show something. Otherwise my api-fetch doesn't render information...